abci/types/application.go

```diff
@@ -63,6 +63,26 @@ func (BaseApplication) Query(req RequestQuery) ResponseQuery {
 
 func (BaseApplication) InitChain(req RequestInitChain) ResponseInitChain {
 	return ResponseInitChain{}
+	/*
+		This part is for code review. How I update validator list?
```

Comment:
Can you clarify your question? Do you want to update a running chain (https://tendermint.com/docs/app-dev/abci-spec.html#endblock)?

Reply:
I am so sorry. I didn't mean to ask. It's not a question. I wanted to show that how I tested it... ignore the question mark! ;)